packages/astro-og-canvas/src/routing.ts

```diff
@@ -27,21 +27,20 @@ function createOGImageEndpoint({ getSlug = pathToSlug, ...opts }: OGImageRouteCo
       (page) => getSlug(...page) === params[opts.param]
     );
     if (!pageEntry) return new Response('Page not found', { status: 404 });
-    return {
-      body: (await generateOpenGraphImage(
-        await opts.getImageOptions(...pageEntry)
-      )) as unknown as string,
-    };
+
+    return new Response(await generateOpenGraphImage(
+      await opts.getImageOptions(...pageEntry)
+    ));
```

Comment:
Did you test this in `dev` as well as `build`? In the past there used to be inconsistencies there. I know Astro v3 moves to using `Response()` so I think the code here is fine, just wanted to double check so we can report any issues if there are any.

Reply:
Answering my own question: just tested this and works great 🥳